### ISSUE TITLE
Fails on CentOS 7 due to missing libsss_sudo

### ISSUE THREAD

**Issue by ssube** [joshuabaird/puppet-ipaclient#15]
While attempting to use this module on a few new CentOS 7 boxes, I'm running into an error stating that:

```
[root@puppet manifests]# puppet agent -t
...
Error: Execution of `/bin/yum -d 0 -e 0 -y list libsss_sudo` returned 1: Error: No matching Packages to list
Error: /Stage[main]/Ipaclient::Sudoers/Package[libsss_sudo]/ensure: change from absent to present failed: Execution of `/bin/yum -d 0 -e 0 -y list libsss_sudo` returned 1: Error: No matching Packages to list
Notice: /Stage[main]/Ipaclient::Sudoers/Package[sssd]: Dependency Package[libsss_sudo] has failures: true
Warning: /Stage[main]/Ipaclient::Sudoers/Package[sssd]: Skipping because of failed dependencies
```

Not sure if this error is fatal or not, but the command exits non-0, so I'm assuming it is.

Running repoquery suggests that sssd does not depend on libsss:

```
[root@puppet manifests]# repoquery --requires --recursive --resolve sssd | grep libsss
libsss_idmap-0:1.11.Z-68.el7_0.5.x86_64
```

And no package libsss_sudo is available in the base repos or EPEL:

```
[root@puppet manifests]# yum search libsss_sudo
...
Warning: No matches found for: libsss_sudo
```

I suspect this is related to the exclusion for Fedora 20+.

Changing line 38 of sudoers.pp to:

```
if (($::operatingsystem == "Fedora" and versioncmp($::operatingsystemrelease, '20') >= 0) or ($::operatingsystem == "CentOS" and versioncmp($::operatingsystemrelease, '7') >= 0)) {
```

appears to resolve the issue. 

Unfortunately, I'm not familiar enough with Ruby or Puppet to know if that's actually right or even reasonable, and haven't been able to thoroughly test whether that dependency is required or not (ran into this while reconfiguring a lab, so will continue testing). I'd be happy to create a pull request if someone can sanity-check the change.

**Comment by stbenjam** [comment]
Thanks! 

Another user opened a PR over @ https://github.com/stbenjam/puppet-ipaclient/pull/13, but it needs a couple of tweaks to make the tests pass.

**Comment by stbenjam** [comment]
Fixed in dd095de
